frontend/index.html

```diff
@@ -11,138 +11,157 @@
                 <li><a href="#decorations">Decorations</a></li>
                 <li><a href="#garden">Garden</a></li>
                 <li><a href="#kitchen">Kitchen</a></li>
                 <li><a href="#tools">Tools</a></li>
                 <li><a href="#storage">Storage</a></li>
             </ul>
+            <ul class="categories-list">
+                <li class="hideOnMobile"><a href="#decorations">Decorations</a></li>
+                <li class="hideOnMobile"><a href="#garden">Garden</a></li>
+                <li class="hideOnMobile"><a href="#kitchen">Kitchen</a></li>
+                <li class="hideOnMobile"><a href="#tools">Tools</a></li>
+                <li class="hideOnMobile"><a href="#storage">Storage</a></li>
+            </ul>
         </nav>
     </header>
 
 
     <!-- Key features  -->
     <section id="features" class="subsection">
         <div class="features container">
-            <div><img src="./assets/images/bag-check.svg" alt=""> Free click & collect</div>
+            <div><img src="./assets/images/bag-check.svg" alt="">Free click & collect</div>
             <div><img src="./assets/images/cart.svg" alt=""> 90 day return policy</div>
             <div><img src="./assets/images/truck.svg" alt=""> Free delivery on 1000s of products</div>
         </div>
     </section>
 
+    <main class="main-content">
+
     <!-- Banner image -->
     <section id="banner">
-        <div class="banner container"><img src="./assets/images/6456-LEYLAND-Web_1400x400px_AW.webp" alt="banner image"></div>
+        <div class="banner container"><img class="banner-image" src="./assets/images/6456-LEYLAND-Web_1400x400px_AW.webp" alt="banner image"></div>
     </section>
 
     <!-- Popular categories -->
     <section id="popular">
         <h2 class="subtitle">Popular categories</h2>
-        <div class="popcategories container">
-        <div class="popular grid">
-            <div>
-               <div><img class= "category-image" src="./assets/images/categories.webp" alt="picture of a chair"></div>
+        <div class="popular grid mobile-grid">
+            <a href="#category-1">
+               <div><img class= "site-image" src="./assets/images/categories.webp" alt="picture of a chair"></div>
                <div>Category 1</div>
-            </div>
-            <div>
-                <div><img class= "category-image" src="./assets/images/categories.webp" alt="picture of a chair"></div>
+            </a>
+            <a href="#category-2">
+                <div><img class= "site-image" src="./assets/images/categories.webp" alt="picture of a chair"></div>
                 <div>Category 2</div>
-             </div>
-             <div>
-                <div><img class= "category-image" src="./assets/images/categories.webp" alt="picture of a chair"></div>
+            </a>
+             <a href="#category-3">
+                <div><img class= "site-image"" src="./assets/images/categories.webp" alt="picture of a chair"></div>
```

Comment:
A detail but it looks like we have an extra space before the class here, and a double `""` in the end of it.